azure-pipelines.yml

```diff
@@ -5,10 +5,10 @@ jobs:
 
   variables:
     GOBIN:  '$(GOPATH)/bin' # Go binaries path
-    GOROOT: '/usr/local/go1.13' # Go installation path
+    GOROOT: '/usr/local/go1.14' # Go installation path
     GOPATH: '$(system.defaultWorkingDirectory)/gopath' # Go workspace path
     modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
-    # TODO: modules should be the default in Go 1.13, so this won't be needed
+    # TODO: modules should be the default in Go 1.14, so this won't be needed
```

Comment:
That's right, I don't think you need to specify it here.

P.S. If you want some help to set up CI with GitHub Actions, let me know 👍 

See https://github.com/caddyserver/caddy/blob/master/.github/workflows/ci.yml

Reply:
Cleaned up that old stuff.
Interesting, I think we can migrate to Github Actions
We need to think as well as how to reduce building steps in the project.
Using xcaddy would improve it a bit, but it would be nice if we can remove all Dockerfiles from this project as well.